Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Building the image
+# 	docker build -t gauge-taiko .
+# Running the image
+# 	docker run  --rm -it -v ${PWD}/reports:/gauge/reports gauge-taiko
+
+# This image uses the official node base image.
+FROM node
+ 
+# The Taiko installation downloads and installs the chromium required to run the tests. 
+# However, we need the chromium dependencies installed in the environment. These days, most # Dockerfiles just install chrome to get the dependencies.
+RUN apt-get update \
+     && apt-get install -y wget gnupg ca-certificates \
+     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+     && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+     && apt-get update \
+     && apt-get install -y google-chrome-stable
+ 
+# Set a custom npm install location so that Gauge, Taiko and dependencies can be 
+# installed without root privileges
+ENV NPM_CONFIG_PREFIX=/home/gauge/.npm-packages
+ENV PATH="${NPM_CONFIG_PREFIX}/bin:${PATH}"
+
+# Add the Taiko browser arguments
+ENV TAIKO_BROWSER_ARGS=--no-sandbox,--start-maximized,--disable-dev-shm-usage
+ENV headless_chrome=true
+ENV TAIKO_SKIP_DOCUMENTATION=true
+#ENV TAIKO_SKIP_CHROMIUM_DOWNLOAD=true
+#ENV TAIKO_BROWSER_PATH=/usr/bin/google-chrome
+ 
+# Add test code
+ADD . /gauge
+
+# Set working directory
+WORKDIR /gauge
+ 
+# Copy the local working folder
+COPY . .
```

Comment:
Do we need both [`ADD`](https://github.com/getgauge/template-js/pull/2/files#diff-3254677a7917c6c01f55212f86c57fbfR31) and `COPY`?